Appraisals

```diff
@@ -2,8 +2,8 @@ appraise 'rails32' do
   gem 'rails', '~> 3.2.0'
 end
 
-appraise 'rails40' do
-  gem 'rails', '~> 4.0.0'
+appraise 'rails42' do
+  gem 'rails', '~> 4.2.0'
```

Comment:
Seems like the easier approach is to have a build matrix and exclude incompatible Rails and Ruby versions on a case by case basis

Reply:
i.e. this is a change which falls outside the scope of this pull request.